src/main/java/org/jabref/logic/bibtex/comparator/MetaDataDiff.java

```diff
@@ -5,11 +5,24 @@
 import java.util.Objects;
 import java.util.Optional;
 
-import org.jabref.logic.l10n.Localization;
 import org.jabref.model.metadata.MetaData;
 import org.jabref.preferences.PreferencesService;
 
 public class MetaDataDiff {
+    public enum Difference {
+        PROTECTED,
```

Comment:
I would extend the enum here and add the localization as second parameter so you don't need a switch statement and instead you can directly return the message

Reply:
That would contradict the whole meaning of this commit, to extract the call to the logic package from the model.

Reply:
But there is something odd

Reply:
Correcting myself: to reduce the calls to the Localization package, to be able to move the 'mother' class to the model package.

Reply:
Then introduce a map with enum keys and l10n values